assets/setup-wizard/style.scss

```diff
@@ -1,394 +1,152 @@
+			border: none;
 		}
 
-		.components-modal__header {
-			padding: 25px 0;
-			margin: 0;
-			height: auto;
-			position: sticky;
-			background: #fff;
-
-			.components-modal__header-heading {
-				font-size: 20px;
-			}
+		&--secondary {
+			color: #101517;
+			border: solid 1px currentColor;
+			background-color: transparent
 		}
 
-		.sensei-list {
-			margin-top: 0;
+		&--link {
+			padding: 0;
+			border: none;
+			background: transparent;
+			text-decoration: underline;
 		}
 
-		.sensei-list__item {
-			border-bottom: 1px solid $gray-5;
+		&:disabled,
+		&[disabled] {
+			opacity: 0.5;
+			cursor: not-allowed;
 		}
 	}
 
-	&__features-installation-feedback {
-		.sensei-list {
-			margin-top: 0;
+	// Actions.
+	&__action-skip {
+		padding-top: 18px;
+		display: block;
 
-			&__item-inner {
-				align-items: flex-start;
-			}
-		}
-
-		.components-spinner {
-			margin: 0;
-		}
-
-		.sensei-setup-wizard__error-message {
-			margin: 0;
-			color: $alert-red;
-
-			.sensei-setup-wizard__retry-button {
-				color: inherit;
-				background: transparent;
-				border: 0;
-				padding: 0;
-				text-decoration: underline;
-				cursor: pointer;
-			}
-		}
-
-		.sensei-setup-wizard__icon-status {
-			padding-top: 2px;
-		}
-
-		.sensei-setup-wizard__circle-icon-wrapper {
-			display: inline-block;
-			width: 18px;
-			height: 18px;
-			line-height: 18px;
-			text-align: center;
-			color: $white;
-			border-radius: 100%;
-			font-style: normal;
-
-			&.success-icon-wrapper {
-				background-color: $primary;
-			}
-			&.error-icon-wrapper {
-				background-color: $alert-red;
-			}
-			&.alert-icon {
-				&::before {
-					display: inherit;
-					width: inherit;
-					height: inherit;
-					content: '!';
-				}
-			}
+		a {
+			font-size: 14px;
 		}
 	}
 
-	&__item-list {
-		.sensei-list__item-title {
-			font-weight: 500;
-		}
-
-		.sensei-list__item-inner {
-			padding-left: 0;
-			padding-right: 0;
-		}
-
-		.sensei-list__item-content {
-			margin-top: 0;
-		}
+	&__submit-error {
+		margin-bottom: 32px;
+	}
+}
 
-		.sensei-list__item-after {
-			.components-button {
-				width: 205px;
-				margin-left: 10px;
-				justify-content: center;
-			}
+// Background animation with the books illustration.
+.sensei-setup-wizard {
+	background-image: url("../images/onboarding-bottom-left-illustration.png"), url("../images/onboarding-top-right-illustration.png");
+	background-position: -346px 100%, calc( 100% + 282px ) 0;
+	background-repeat: no-repeat;
+	transition: 0.5s background-position;
+	background-size: auto 220px;
 
-			.sensei-setup-wizard__error-message {
-				width: 205px;
-				margin-left: 10px;
-				padding: 6px;
-				color: red;
-				font-size: 12px;
-			}
-		}
+	@media (min-width: 1100px) and (min-height: 700px) {
+		background-size: auto;
 	}
 
-	&__mailinglist-signup-form {
-		.sensei-list__item-text {
-			flex: 1;
-		}
-		.components-base-control__field {
-			margin-bottom: 0;
-		}
+	@at-root {
+		.sensei-setup-wizard-page--welcome & {
```

Comment:
It's here (in a generic area) because later we will want to add it for the last step too.